spring-components/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.rdf4j</groupId>
+		<artifactId>rdf4j</artifactId>
+		<version>3.7.0</version>
```

Comment:
Shouldn't the version be 3.6.1-SNAPSHOT? At least, that's what the rest of the for is on...

Reply:
er... that should have read, 'the rest of the fork'

Reply:
Probably it will need to be 4.0.0-develop after merging...

Reply:
Good point! It should be 4.0.0-SNAPSHOT. Since this is a new pom file, you will need to fix this as part of this PR, before merge. Probably best to do a rebase with the develop branch first, to get your feature branch up to date.